### DISCUSSION TITLE
when i open mcap file that convert from ros1 bag by plotjunggler, i went wrong

### DISCUSSION THREAD

**Discussion by 123dbl** [foxglove/mcap#1112] [General]
with exception blow:

The plugin [DataLoad MCAP] thrown the following exception: 

 No parsing available for encoding [ros1msg] nor [ros1

**Comment by defunctzombie** [comment]
I'd recommend starting by posting question to PlotJugger: https://github.com/facontidavide/PlotJuggler

I recommend you provide more information like the exact steps you took and possibly the file so the PlotJuggler maintainers can take a look.
